### ISSUE TITLE
Updated disallowed-blocks

### ISSUE THREAD

**Issue by xSimplified-x** [EngineHub/WorldEdit#1696]
Update the disallowed-blocks with new blocks that were added in newer versions, you can't use //set fire, but you can use //set soul_fire

**Comment by wizjany** [comment]
er, that's a config option, you can change it yourself as you see fit.

**Comment by xSimplified-x** [comment]
Yes, but the default config should be updated with the newer blocks that were added

**Comment by octylFractal** [comment]
We're more likely to remove blocks from the list at this point, as e.g. `fire` is hard to type by accident now. So far the only candidate for a revised list is `grass` (easy to use instead of `grass_block`).

Closing for now, might revisit updating the list later.
